packages/core/src/indexing/database-index-api.ts

```diff
@@ -118,10 +119,8 @@ export abstract class DatabaseIndexApi<DateType = Date | number> {
       .first('indices')
       .where('model', model.toString())
 
-    if (res) {
+    if (res && res.indices) {
       return JSON.parse(res.indices)
-    } else {
-      undefined
     }
```

Comment:
instead of falling through and implicitly returning `undefined`, can we change this to explicitly return `null` if there are no indices?  Or maybe just an empty array would also be fine?

Reply:
Empty array didn't work, because there are cases where we want to know if there are actually indices specified. Can change to explicit return.